src/absil/il.fs

```diff
@@ -13,18 +13,12 @@ open Microsoft.FSharp.Compiler.AbstractIL
 open Microsoft.FSharp.Compiler.AbstractIL.Internal
 open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library
 open Microsoft.FSharp.Compiler.AbstractIL.Diagnostics
-open System
 open System.Collections
 open System.Collections.Generic
 open System.Collections.Concurrent
  
 let logging = false 
 
-let runningOnWindows =
```

Comment:
Hmmm.. basically we don't want to be throwing exceptions in ilwrite.fs in FCS (the only place this is used) if we can detect that we are on Windows, .NET Framework.  If you could find some way to avoid relying on the try/catch in ilwrite.fs when on .NET Framework that would be great. Thanks